datadog_sync/utils/resource_utils.py

```diff
@@ -92,6 +92,37 @@ def give_up_diffing(self, level, diff_instance) -> bool:
         return False
 
 
+def create_global_downtime(config: Configuration):
+    """Create global downtime"""
+    payload = {
+        "data": {
+            "attributes": {
+                "message": "Downtime created by datadog-sync-cli to mute all monitors synced. \
+                    To be manually removed after failover is comeplete.",
```

Comment:
"To be manually removed after failover is comeplete" -> "To be manually removed during failover when monitors have enough telemetry to trigger appropriately"